android/src/main/java/com/rnmaps/maps/MapManager.java

```diff
@@ -193,7 +193,6 @@ public void setMapPadding(MapView view, @Nullable ReadableMap padding) {
         }
 
         view.applyBaseMapPadding(left, top, right, bottom);
-        view.map.setPadding(left, top, right, bottom);
```

Comment:
`map.setPadding()` is already called during `applyBaseMapPadding()` so this line seems unnecessary.